src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -31,7 +32,7 @@ def fdb_vlanmac(self, fdb):
                 self.bvid_vlan_map[fdb["bvid"]] = vlan_id
         else:
             return None
-        if not isinstance(vlan_id, str):
+        if not isinstance(vlan_id, bytes) and not isinstance(vlan_id, str):
```

Comment:
why is this change required?

Reply:
I fixed it in above line, and this is not required anymore.

---
In reply to: [594597739](https://github.com/Azure/sonic-snmpagent/pull/203#discussion_r594597739) [](ancestors = 594597739)